tools/export/mcuxpresso/LPC55S69_NS_cproject.tmpl

```diff
@@ -0,0 +1,29 @@
+{% extends "mcuxpresso/.cproject.tmpl" %}
```

Comment:
Is it really necessary to escape all the XML content in this way? Other templates in the same directory are not escaped in this way (see .cproject.tmpl for example). If the escaping can be removed then this would be much more readable.